### PR TITLE
Run tests on windows

### PR DESCRIPTION
**Pull request by mitchellhamilton**

### CODE REVIEW

File: packages/cli/src/rollup-plugins/typescript-declarations/create-generator.ts [hunk 1]
```diff
@@ -164,7 +165,7 @@ export async function createDeclarationCreator(
               if (
                 !allDeps.has(resolvedModule.resolvedFileName) &&
                 !resolvedModule.isExternalLibraryImport &&
-                resolvedModule.resolvedFileName.includes(dirname)
+                resolvedModule.resolvedFileName.includes(normalizedDirname)
```
Comment (Andarist) [inline packages/cli/src/rollup-plugins/typescript-declarations/create-generator.ts:168]: @RafalFilipek this should basically fix your issue 😃

Comment (RafalFilipek) [reply packages/cli/src/rollup-plugins/typescript-declarations/create-generator.ts:168]: I'll tattoo this line on my back if it's true :D

Comment (Andarist) [reply packages/cli/src/rollup-plugins/typescript-declarations/create-generator.ts:168]: Better start looking for a tattoo artist 😁

**Comment by mitchellhamilton** [comment]
Thanks for this @Andarist!!

**Pull request merged**